apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss

```diff
@@ -64,6 +74,7 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 }
 
 .page-template-modal__inner {
+	position: relative;
 	max-width: 720px;
```

Comment:
I'd suggest removing this max-width rule, to take advantage of the whole screen width independently of the current viewport size.

**before**
![image](https://user-images.githubusercontent.com/77539/65686231-4614bd00-e03a-11e9-8158-84107c186bfd.png)


**after**
![image](https://user-images.githubusercontent.com/77539/65686200-36957400-e03a-11e9-928e-6cced1308771.png)

Reply:
It's a good point. I _think_ I've already done this.